crawler.py

```diff
@@ -301,6 +311,10 @@ def start_driver(self):
                                                  'temporary': True})
             time.sleep(2)
 
+            # install a parallel extension to run alongside pb
+            if self.load_extension:
+                self.driver.install_addon(self.load_extension, temporary=True)
```

Comment:
Let's load Firefox zip extension packages, to match what we do in Chrome. So, no `temporary` flag.